specification/security/resource-manager/Microsoft.Security/preview/2020-08-06-preview/iotSensors.json

```diff
@@ -0,0 +1,285 @@
+          }
+        },
+        "tags": [
+          "IoT Sensors"
+        ],
+        "operationId": "IotSensors_Delete",
+        "description": "Delete IoT sensor",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSensorScope"
+          },
+          {
+            "$ref": "#/parameters/IotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Deleted IoT sensor"
+          },
+          "204": {
+            "description": "IoT sensor does not exist"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.Security/iotSensors/{iotSensorName}/downloadActivation": {
+      "post": {
+        "x-ms-examples": {
+          "Download activation file": {
+            "$ref": "./examples/IotSensors/DownloadActivation.json"
+          }
+        },
+        "tags": [
+          "IoT Sensors"
+        ],
+        "operationId": "IotSensors_DownloadActivation",
+        "description": "Download sensor activation file",
+        "parameters": [
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/IotSensorScope"
+          },
+          {
+            "$ref": "#/parameters/IotSensorName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Downloaded activation file"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../common/v1/types.json#/definitions/CloudError"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "IotSensorsList": {
+      "description": "List of IoT sensors",
+      "type": "object",
+      "properties": {
+        "value": {
+          "description": "List data",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/IotSensor"
+          }
+        }
+      }
+    },
+    "IotSensor": {
+      "type": "object",
+      "description": "IoT sensor",
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "IoT sensor properties",
+          "$ref": "#/definitions/IotSensorProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "IotSensorProperties": {
+      "type": "object",
+      "description": "IoT sensor properties",
+      "properties": {}
```

Comment:
this is weird, Is this resource type currently in test? if so, why do you have it in public repo. empty properties dont make sense.